tasks/letsencrypt.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: install gnupg2
+  apt:
+    name: gnupg2
+    state: present
+    cache_valid_time: 3600
+
+- name: download the certbot-auto signature file
+  get_url:
+    url: https://dl.eff.org/certbot-auto.asc
+    dest: /tmp/certbot-auto.asc
+    force: yes
+
+- name: install the certbot-auto script
+  get_url:
+    url: https://dl.eff.org/certbot-auto
+    dest: /usr/local/bin/certbot-auto
+    force: yes
+    mode: 755
+
+- name: receive the certbot gpg key
+  command: gpg2 --keyserver pgpkeys.mit.edu --recv-key A2CFB51FA275A7286234E7B24D17C995CD9775F2
+
+- name: verify the certbot-auto download
+  command: gpg2 --trusted-key 4D17C995CD9775F2 --verify /tmp/certbot-auto.asc /usr/local/bin/certbot-auto
```

Comment:
IMO it is appropriate to delete /usr/local/bin/certbot-auto when verification fails so that the cron job isn't running theoretically untrusted code (as root no less).